src/pages/Cart/Cart.jsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import './Cart.scss';
+// import img from '../../../public/images/cart_sample.jpg';
+
+const Cart = () => {
+  return (
+    <div className="cart">
+      <div className="cartArea">
+        <form className="formArea">
+          <div className="cartHeader">
+            <div className="formInput">
+              <label htmlFor="checkAll" className="formLabel">
+                <span>모두 선택</span>
+                <span>(2/2)</span>
+              </label>
+              <input type="checkbox" id="checkAll" className="formCheck" />
+            </div>
+            <div className="btnArea">
+              <button type="button" className="btn btnText">
+                <span>선택삭제</span>
+              </button>
+            </div>
+          </div>
+          <div className="cartBody">
+            <ul className="listArea">
+              <li className="listItem">
+                <strong className="listTitle">예술주조</strong>
+                <div className="listDetail">
+                  <div className="detailArea">
+                    <div className="formInput">
+                      <label htmlFor="checkAll" className="formLabel blind">
+                        <span>선택</span>
+                      </label>
+                      <input
+                        type="checkbox"
+                        id="checkAll"
+                        className="formCheck"
+                      />
+                    </div>
+                    <div className="detailInfo">
+                      <div className="detailImage">
+                        <img
+                          src="../../../public/images/cart_sample.jpg"
```

Comment:
public 폴더는 절대경로로 접근하시면 됩니다.
```suggestion
                          src="/images/cart_sample.jpg"
```